### DISCUSSION TITLE
Account suspend

### DISCUSSION THREAD

**Discussion by DrKingsuk** [community/community#110680] [New to GitHub] [answered]
### Select Topic Area

Question

### Body

Hi i am a new user in github i try to deploy my project in vercel. Throw github account after this my github account suspend.i content to github team my token id 2638729 they did provide any information about that they just say Your account has been hidden from view, as it appears to have been one of many created in an account farming effort. I don't understand that plz help

**Comment by elhizazi1** [comment]
Response

**Comment by DrKingsuk** [comment]
Means

**Comment by queenofcorgis** [comment, marked as answer]
Thanks for posting in the GitHub Community, @DrKingsuk ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions.

Given your inquiry, the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
